travis-ci/run-redeployment-tests.sh

```diff
@@ -0,0 +1,28 @@
+set -e
+
+export AWS_ACCESS_KEY_ID="$INTEGRATION_AWS_ACCESS_KEY_ID"
+export AWS_SECRET_ACCESS_KEY="$INTEGRATION_AWS_SECRET_ACCESS_KEY"
+export AWS_DEFAULT_REGION="$INTEGRATION_AWS_DEFAULT_REGION"
+
+if [ -z "$DEPLOYMENT" ]; then
+  DEPLOYMENT=$(node ./travis-ci/select-stack.js)
+  if [ "$DEPLOYMENT" = "none" ]; then
+    echo "Unable to determine integration stack" >&2
+    exit 1
+  fi
+fi
+export DEPLOYMENT
+
+(
+  cd example
+  if [ "$USE_NPM_PACKAGES" = "true" ]; then
+    yarn
+  else
+    (cd .. && ./bin/prepare)
+  fi
+
+  yarn redeploy-test
+)
+RESULT="$?"
```

Comment:
Catching the result and then explicitly returning it are not necessary.  Since the subshell is the last statement, its return value will be the return value of the entire script.